### ISSUE TITLE
Exclude Z or otherwise make date in `LIVE_CONTENT` accurate

### ISSUE THREAD

**Issue by vinnyjth** [ApollosProject/apollos-prototype#490]

**Comment by redreceipt** [comment]
@vinnyjth can I get a little more context here? Or do you wanna explain over a call?

**Comment by redreceipt** [comment]
@vinnyjth where do I go from here to test out that API call?

![screen shot 2019-02-05 at 3 30 12 pm](https://user-images.githubusercontent.com/2659478/52302399-0b4eff00-295b-11e9-912d-8a6fa236dea7.png)

**Comment by vinnyjth** [comment]
@redreceipt `ContentChannelItems` -> GET `/api/ContentChannelItems`

**Comment by redreceipt** [comment]
Just an update, talking to John E from Spark about whether Rock supports inputting data with timezone info so we don't have to do any funky conversions. The example API response has a "Z" which is leading me to believe it does support it but for some reason, our content doesn't have it. Will update as I find out more.

**Comment by redreceipt** [comment]
@conrad-vanl @vinnyjth hoping that scheduled time with a Rock expert happens. I can confidently move forward on this task as soon as someone knowledgable with Rock data and APIs answers a couple simple questions.

Waiting on the answer to these questions: Why does our data not have TZ info in it when the example API response does? Do we have to enable it somewhere for it to apply during content creation?

Some context:
Example response from API: `2019-02-05T21:44:37.539Z`
Our data: `2019-02-05T21:44:37.539`
...so a filter won't be consistent
Our data needs to be: `2019-02-05T21:44:37.539Z` or `2019-02-05T21:44:37.539-5:00`

If you can't input TZ info by default, I think the best solution here is two options:

1. Write our own content data API call that adds TZ info to data before the filter. It's practically the same with the addition of a couple of characters based on what the local Rock time zone is set to. The only drawback is you are assuming _all_ content has the same time zone.
2. Write a plugin that gives the user the ability to input TZ info when content is created.

Thoughts???

**Comment by conrad-vanl** [comment]
@redreceipt we brought this up during our call and Jon E. asked us to send them the exact endpoints / responses that we are getting to show what's not working

**Comment by redreceipt** [comment]
@conrad-vanl What's the best for me to send them? Does he have an email address I should use? Does that need to come from you?

**Comment by redreceipt** [comment]
From Jon: 

![Screen Shot 2019-03-27 at 11 46 11 AM](https://user-images.githubusercontent.com/2659478/55090732-facb2300-5085-11e9-9c04-94fa0565a2d6.png)

TL;DR
I think Rock is converting time strings incorrectly...

**Comment by vinnyjth** [comment]
Done in #611
